DQM/HLTEvF/plugins/LumiMonitor.cc

```diff
@@ -181,44 +176,44 @@ void LumiMonitor::bookHistograms(DQMStore::IBooker& booker,
                                lumi_binning_.xmin,
                                lumi_binning_.xmax);
   me->setAxisTitle("LS", 1);
-  me->setAxisTitle("scal inst lumi E30 [Hz cm^{-2}]", 2);
+  me->setAxisTitle("online inst lumi E30 [Hz cm^{-2}]", 2);
   histograms.lumiVsLS = me;
 
   me = booker.bookProfile("puVsLS",
-                          "scal PU vs LS",
+                          "online PU vs LS",
```

Comment:
could you also change `PU` to `pileup` and `LS` to `lumisection` in the labels ?